test/e2e/node_maintenance_test.go

```diff
@@ -179,6 +179,9 @@ var _ = Describe("Starting Maintenance", func() {
 			// it should be caused by the test deployment's termination graceperiod > drain timeout
 			Expect(getOperatorLogs()).To(ContainSubstring(nodemaintenance.FixedDurationReconcileLog))
 
+			//validate that operator controller pod have app.kubernetes.io/name label
+			Expect(validateOperatorCustomLabels()).To(BeTrue(), "operator custom label validation failed")
```

Comment:
the return value and checking it isn't needed IMHO, everything is checked inside the function already?